packages/react-core/src/components/Modal/ModalContent.tsx

```diff
@@ -7,6 +7,7 @@ const FocusTrap: any = require('focus-trap-react');
 
 import styles from '@patternfly/react-styles/css/layouts/Bullseye/bullseye';
 import titleStyles from '@patternfly/react-styles/css/components/Title/title';
+import modalStyles from '@patternfly/react-styles/css/components/ModalBox/modal-box';
```

Comment:
is this import used?

Reply:
Nope, and I don't remember adding it. Good catch.